From: Eric Desrochers <slashd@ubuntu.com>
Date: Wed, 19 Aug 2020 22:49:24 +0000
Subject: Change default location to /tmp

Forwarded: not-needed
---
 sos.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sos.conf b/sos.conf
index 2be22a3..26778d2 100644
--- a/sos.conf
+++ b/sos.conf
@@ -7,6 +7,7 @@
 #verify = yes
 #batch = yes
 #log-size = 15
+tmp-dir = /tmp
 
 [report]
 # Options that will apply to any `sos report` run should be listed here.
